### ISSUE TITLE
Jinshi-Ben meeting 2019-11-12 #26

### ISSUE THREAD

**Issue by jinshijian** [PNNL-TES/TES-mgmt#26]
# Finished this week
## GlobalC 
* Renamed lots of functions, notations made for clarity.
* Created a new markdown file to compare 3 resample scenarios (not use SD, SD = m*cv when missing, and SD = maximum reported SD).
* The updates are available in the branch now, but I cannot merge it to the master branch (due to conflict again). 

## Surrogates
* In addition to R2, I also compared residuals, EMSE, d, and EF (see Yang et al, 2014. An evaluation of the statistical methods for testing the performance of crop models with observed data. Agricultural Systems, 127: 81-89. )
* Tm as a surrogate of Tsoil residual comparison
![image](https://user-images.githubusercontent.com/13302161/68610956-02113680-0487-11ea-8fe4-9ebe75dc0510.png)
* Pm as a surrogate of Tair residual comparison
![image](https://user-images.githubusercontent.com/13302161/68611068-3d136a00-0487-11ea-8a6c-fc814ec154ac.png)
* Tm as a surrogate of Tsoil (RMSE, d, and EF comparison)
![image](https://user-images.githubusercontent.com/13302161/68611142-692eeb00-0487-11ea-8651-8c29093eb3bd.png)
* Pm as a surrogate of SWC (RMSE, d, and EF comparison)
![image](https://user-images.githubusercontent.com/13302161/68611197-911e4e80-0487-11ea-83ec-966fa736e4c7.png)
* In conclusion: there are no differences between residual, RMSE, d, or RMSE when using Tair to predict Rs comparing with Tsoil to predict Rs, same for Precipitation as a surrogate of SWC (by visualization, should do some statistics?).
* Now there are too many figures, I am wondering whether there are better ways to present the results.
* I got an error when knit: function gather is defunct in drake.
* I also got some questions on drake need to discuss.

## SRDB
* Finished "time and depth" check, working on updating now.
* Checked some srdb new papers input, will discuss with Darlin and Jason tomorrow.
